R/learn.R

```diff
@@ -119,9 +122,12 @@ mssql_learn_query <- function() { # taken directly from {datamodelr}
       and ind_col.column_id = cols.column_id
     left outer join sys.systypes [types] on
       types.xusertype = cols.system_type_id
+  where tabs.schema_id = schema_id('%s')
   order by
     tabs.create_date,
-    cols.column_id"
+    cols.column_id",
+  gsub("(^|[^'])(['])($|[^'])", "\\1\\2\\2\\3", schema)
```

Comment:
```suggestion
  DBI::dbQuoteString(con, schema)
```

Reply:
Ok, thanks, that looks way better :) Only thing is, that it also doubles `''`, so if the user tries to use their SQL knowledge, it will fail. But I think that's a minor issue.